File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_Three/Python/prog5.py
num1, num2, num3= [int(i) for i in input("Input : ").split()]
if(num1 >= num2 and num1 >= num3):
    print("Max : ", num1)
elif(num2 >= num3):
    print("Max : ", num2)
else:
    print("Max : ", num3)

File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_Three/C++/prog5.cpp
#include<iostream>

int main() {
	int num1, num2, num3;
	std::cout << "Enter num1 : ";
	std::cin >> num1;
	std::cout << "Enter num2 : ";
	std::cin >> num2;
	std::cout << "Enter num3 : ";
	std::cin >> num3;
	if(num1 >= num2 && num1 >= num3) {
		std::cout << "Max = " << num1;
	} else if(num2 >= num3) {
		std::cout << "Max = " << num2;
	} else {
		std::cout << "Max = " << num3;
	}
	return 0;
}
